libraries/AP_HAL_ChibiOS/hwdef/include/minimize_features.inc

```diff
@@ -52,3 +52,6 @@ define OSD_PARAM_ENABLED 0
 
 # no Winch if minimized:
 define AP_WINCH_ENABLED 0
+
+# prune out some odd camera backends:
+define AP_CAMERA_MOUNT_ENABLED 0
```

Comment:
this is fine but we could also make the ap-camera-mount-enabled depend upon ap-camera && hal-mount-enabled.  Just an idea though.

Reply:
I think we did - so yes, this bit is redundant *except* if you add mount back in then you still won't get this backend- which I think is fine?

Reply:
yes, it's fine to have a mount without the AP_Camera library..